src/contracts/pods/EigenPodManagerStorage.sol

```diff
@@ -64,6 +64,8 @@ abstract contract EigenPodManagerStorage is IEigenPodManager {
      */
     mapping(address => int256) public podOwnerShares;
 
+    bool public isDeneb;
+
```

Comment:
gotta fix the gap size from this new variable

Reply:
fixed!